stages/org.osbuild.systemd-journald

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python3
+"""
+Configures systemd-journald
+
+The 'config' option allows to create a systemd-journald configuration drop-in
+file in `/usr/lib/systemd/journald.conf.d` with the name `filename`.
+
+Drop-in configuration files can specify the following subset of options:
+- 'Journal' section
+  -  'Storage' option
+  -  'Compress' option
+  -  'SplitMode' option
+  -  'MaxFileSec' option
+  -  'MaxRetentionSec' option
+  -  'SyncIntervalSec' option
+  -  'Audit' option
+
+At least one option must be specified in the 'Journal' section.
+"""
+
+import configparser
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA = r"""
+"additionalProperties": false,
+"properties": {
+    "filename": {
+        "type": "string",
+        "description": "Name of the systemd-journald drop-in file.",
+        "pattern": "^([[a-zA-Z0-9_]|\\.|-]){1,250}\\.conf$"
```

Comment:
I am not sure I understand what this regexp is supposed to match on. Can you explain? The nested brackets look fishy. How about this:

```suggestion
        "pattern": "^[a-zA-Z0-9_\\.-]{1,250}\\.conf$"
```

Reply:
I'm currently changing this since I had a few mishaps with the double backlashes, the current version is: `^([a-zA-Z0-9_]|\\.|-){1,250}\\.conf$`. So, alphanumeric characters, dots, - and _ are allowed for the filename, which must end in .conf.

Which is equivalent to what you have commented: `^[a-zA-Z0-9_\\.-]{1,250}\\.conf$, so I'll update it.